src/engine/enginemaster.cpp

```diff
@@ -260,47 +266,78 @@ void EngineMaster::processChannels(unsigned int* busChannelConnectionFlags,
             continue;
         }
 
-        bool needsProcessing = false;
```

Comment:
oh I see your comment in the PR: "channels are processed, regardless of the signal routing above, to have the VU meter always active".